metadata_watcher/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Dockerfile extending the generic Python image to run an metadata watcher app.
+
+FROM gcr.io/google_appengine/python-compat
```

Comment:
Please never ever use python-compat.

Please use python, preferably with Python 3: https://github.com/GoogleCloudPlatform/python-runtime

Also note that the Python runtime are quite big, I would avoid Python for sidecars unless really necessary.